[Finishes #187355085] Eslint configuration

Celse13 opened the pull request:
## What does this PR do?
Configure ESLint and Git Pre commit hook to run tests and ESlint format
## Description of Task to be completed?
This task was about to run Eslint, test, and format before the pre-commit
## How should this be manually tested?
You need to follow the common practice of pushing to the remote repositiory, by git add, git commit -m 'Add message' after that the Eslint will run lint and format to fix the indentation issue and make sure that the code follow ES6 guidelines
## Any background context you want to provide?
## What are the relevant pivotal tracker/Trello stories?
Configure ESLint and Git Pre commit hook to run tests and ESlint format https://www.pivotaltracker.com/story/show/187355085
## Screenshots (if appropriate)
![Screenshot 2024-04-05 160144](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/117368635/640dc687-64e1-40fe-a13f-60f90d85c421)

## Questions

UmuhireJessie reviewed (changes requested):
@Celse13 well done! Please do the following:
1. Squash the commits, 1 commit should be added to `develop`
2. Rebase to `develop`
3. Resolve the conflicts
4. Change the commit message, Refer to this [https://www.pivotaltracker.com/help/articles/github_integration/](https://www.pivotaltracker.com/help/articles/github_integration/)

Celse13 committed:
[Finishes #187355085] Eslint configuration

Added lin

Add

m

Add test

[Finishes #187355085] Eslint configuration

Adde tst

Adding

Add

kd

A

Adding

 add

Add

a

 ad

[Finishes #187355085] Eslint configuration

Adding some indentation

Adding some change

[Finishes #187355085] Eslint configuration

[Finishes #187355085] Eslint configuration

[Finishes #187355085] Eslint configuration

adding changes

adding changes

Update README.md

Update README.md

UmuhireJessie reviewed (approved):
Good!
